v3/linworks/fave60/fave60.json

```diff
@@ -0,0 +1,94 @@
+{
+    "name":"Linworks Fave 60",
+    "vendorId":"0x4C58",
+    "productId":"0x000D",
+    "keycodes": ["qmk_lighting"],
+    "menus": 
+            [
+              {
+                "label": "Lighting",
+                "content": [
+                  {
+                    "label": "Underglow",
+                    "content": [
+                      {
+                        "label": "Brightness",
+                        "type": "range",
+                        "options": [0, 255],
+                        "content": ["id_qmk_rgblight_brightness", 2, 1]
+                      },
+                      {
+                        "label": "Effect",
+                        "type": "dropdown",
+                        "content": ["id_qmk_rgblight_effect", 2, 2],
+                        "options": [
+                          "All Off",
+                          "Solid Color",
+                          "Knight 1",
+                          "Knight 2",
+                          "Knight 3",
+                          "Rainbow Mood 1",
+                          "Rainbow Mood 2",
+                          "Rainbow Mood 3",
+                          "Rainbow Swirl 1",
+                          "Rainbow Swirl 2",
+                          "Rainbow Swirl 3",
+                          "Rainbow Swirl 4",
+                          "Rainbow Swirl 5",
+                          "Rainbow Swirl 6",
+                          "Snake 1",
+                          "Snake 2",
+                          "Snake 3",
+                          "Snake 4",
+                          "Snake 5",
+                          "Snake 6",
+                          "Twinkle 1",
+                          "Twinkle 2",
+                          "Twinkle 3",
+                          "Twinkle 4",
+                          "Twinkle 5",
+                          "Twinkle 6"
```

Comment:
Wrong order please refer to QMK docs (for enum values) or below
https://github.com/the-via/reader/blob/master/src/common-menus/qmk_rgblight.ts